integrationTests/vm/wasm/delegation/delegation_test.go

```diff
@@ -33,39 +32,6 @@ var NewBalance = wasm.NewBalance
 var NewBalanceBig = wasm.NewBalanceBig
 var RequireAlmostEquals = wasm.RequireAlmostEquals
 
-func TestDelegation_Upgrade(t *testing.T) {
```

Comment:
why did we remove this test?

Reply:
Explained in comment below. It's a bit hard to obtain a variant of the delegation contract that satisfies the `upgrade` requirement - without additional GitHub / cross-repo / versioning bureaucracy. Confidence of tests remains the same.